test/src/test/testtool/Validators/Schema/SoapValidator.wsdl

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<wsdl:definitions xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:tns="urn:items/wsdl/" xmlns:ns1="http://nn.nl/XSD/Generic/MessageHeader/1" xmlns:ns2="urn:items" targetNamespace="urn:items/wsdl/">
```

Comment:
Wel raar dat nu de messageheader weg is?

Reply:
Hey dat had ik niet bewust aangepast... Zoek ik even uit.

Reply:
IntelliJ zegt dat `ns1` niet gebruikt wordt en zal hem misschien daarom weggehaald hebben?